lib/identity/domain/use_cases/identity/get_private_key_use_case.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:injectable/injectable.dart';
+import 'package:polygonid_flutter_sdk/common/domain/domain_logger.dart';
+import 'package:polygonid_flutter_sdk/common/domain/use_case.dart';
+import 'package:polygonid_flutter_sdk/identity/domain/repositories/identity_repository.dart';
+import 'package:polygonid_flutter_sdk/identity/domain/use_cases/get_current_env_did_identifier_use_case.dart';
+
+class GetPrivateKeyUseCase extends FutureUseCase<String, String> {
```

Comment:
Now we have the UC, we should use it everywhere instead of calling directly `IdentityRepository.getPrivateKey()`
See `AddNewIdentityUseCase` and `CheckIdentityValidityUseCase`